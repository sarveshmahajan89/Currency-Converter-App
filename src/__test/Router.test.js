import React from 'react';

import Router from '../component/Router.js'
import {shallow} from "enzyme/build/index";

describe('Router', function() {
    it('Router renders correctly', () => {

        expect(<Router/>).toMatchSnapshot();
    });
    it('should render Router correctly with shallow', function () {

        const component = shallow(<Router/>);
        expect(component).toMatchSnapshot();
    });

});