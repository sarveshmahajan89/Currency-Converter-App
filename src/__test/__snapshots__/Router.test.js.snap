// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router Router renders correctly 1`] = `<Router />`;

exports[`Router should render Router correctly with shallow 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header
    className="masthead"
  >
    <div
      className="container"
    >
      <div
        className="intro-text"
      >
        <div
          className="content"
        >
          <Switch>
            <Route
              component={[Function]}
              exact={true}
              path="/"
            />
            <Route
              component={[Function]}
              path="/about"
            />
            <Route
              component={[Function]}
              path="/contact"
            />
          </Switch>
        </div>
      </div>
    </div>
  </header>,
  "nodes": Array [
    <header
      className="masthead"
    >
      <div
        className="container"
      >
        <div
          className="intro-text"
        >
          <div
            className="content"
          >
            <Switch>
              <Route
                component={[Function]}
                exact={true}
                path="/"
              />
              <Route
                component={[Function]}
                path="/about"
              />
              <Route
                component={[Function]}
                path="/contact"
              />
            </Switch>
          </div>
        </div>
      </div>
    </header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_didScheduleRenderPhaseUpdate": false,
    "_dispatcher": Object {
      "readContext": [Function],
      "useCallback": [Function],
      "useContext": [Function],
      "useDebugValue": [Function],
      "useDeferredValue": [Function],
      "useEffect": [Function],
      "useImperativeHandle": [Function],
      "useLayoutEffect": [Function],
      "useMemo": [Function],
      "useReducer": [Function],
      "useRef": [Function],
      "useResponder": [Function],
      "useState": [Function],
      "useTransition": [Function],
    },
    "_element": <Router />,
    "_firstWorkInProgressHook": null,
    "_forcedUpdate": false,
    "_instance": Router {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": null,
      "updater": Updater {
        "_callbacks": Array [],
        "_renderer": [Circular],
      },
    },
    "_isReRender": false,
    "_newState": null,
    "_numberOfReRenders": 0,
    "_renderPhaseUpdates": null,
    "_rendered": <header
      className="masthead"
    >
      <div
        className="container"
      >
        <div
          className="intro-text"
        >
          <div
            className="content"
          >
            <Switch>
              <Route
                component={[Function]}
                exact={true}
                path="/"
              />
              <Route
                component={[Function]}
                path="/about"
              />
              <Route
                component={[Function]}
                path="/contact"
              />
            </Switch>
          </div>
        </div>
      </div>
    </header>,
    "_rendering": false,
    "_updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
    "_workInProgressHook": null,
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Router />,
}
`;
